pkg/subctl/operator/common/embeddedyamls/generators/yamls2go.go

```diff
@@ -26,20 +26,19 @@ import (
 )
 
 var files = []string{
-	"crds/submariner.io_submariners_crd.yaml",
-	"crds/submariner.io_servicediscoveries_crd.yaml",
-	"role.yaml",
-	"role_binding.yaml",
-	"service_account.yaml",
-	"submariner/globalnet_cluster_role.yaml",
-	"submariner/globalnet_cluster_role_binding.yaml",
-	"cluster_role.yaml",
-	"cluster_role_binding.yaml",
-	"lighthouse/crds/multiclusterservices_crd.yaml",
-	"lighthouse/crds/serviceexport_crd.yaml",
-	"lighthouse/crds/serviceimport_crd.yaml",
-	"lighthouse/cluster_role_binding.yaml",
-	"lighthouse/cluster_role.yaml",
```

Comment:
Please. let's remain the 

lighthouse/
and 
submariner/

CRDs untouched, or let's move them to a different directory altogether, or see how to do it.

The convention we had here is:

Everything on the 
   * submariner directory is stuff used by the core submariner services
   * lighthouse directory is stuff used by lighthouse
   * <outside> : stuff and CRDs that the operator need to work & exist.

Currently after latest patches, the items in the "submariner & lighthouse" directories are installed by the operator internally, they don't need to be part of the CSV anympre.